client/my-sites/sites/index.jsx

```diff
@@ -47,7 +47,7 @@ class Sites extends Component {
 
 		// Plans are for not Jetpack or Jetpack upgradeable sites.
```

Comment:
I know this isn't part of the changes, but this comment is super confusing. I suggest removing it, as the code is small and easier to grok.